src/components/strategies/edit/overlapping/DepositOverlappingStrategy.tsx

```diff
@@ -65,32 +65,40 @@ export const DepositOverlappingStrategy: FC<Props> = (props) => {
     return oldMarketPrice.toString();
   };
 
-  const setBuyBudget = async (sellBudget: string) => {
-    if (!sellBudget) return order0.setBudget('');
-    const buyBudget = await carbonSDK.calculateOverlappingStrategyBuyBudget(
-      base.address,
-      quote.address,
-      order0.min,
-      order1.max,
-      getMarketPrice(),
-      spreadPPM.toString(),
-      sellBudget
+  const setBuyBudget = async (value: string) => {
```

Comment:
I think it would be more clear if you call it sellBudgetDelta or something explanatory - and not value

Reply:
fixed